flink-core/src/main/java/org/apache/flink/configuration/TaskManagerOptions.java

```diff
@@ -229,15 +239,21 @@
 			key("taskmanager.memory.off-heap")
 			.defaultValue(false)
 			.withDescription("Memory allocation method (JVM heap or off-heap), used for managed memory of the" +
-				" TaskManager as well as the network buffers.");
+				" TaskManager as well as the network buffers. For setups with larger quantities of memory, this can" +
+				" improve the efficiency of the operations performed on the memory .");
```

Comment:
actually, network buffers are always off-heap since 1.5 ([FLINK-7316](https://issues.apache.org/jira/browse/FLINK-7316) must have forgotten to update the docs of this parameter)

Reply:
Given that it is a critical to tune `taskmanager.memory.preallocate` along with this one, should we also point to `taskmanager.memory.preallocate` from here?